src/dune_pkg/workspace.ml

```diff
@@ -52,6 +52,15 @@ module Repository = struct
     }
   ;;
 
+  let binary_dev_tools =
```

Comment:
Does it only have to be just `dev-tools`? From what I understand, nothing (but relocatability) is preventing us from shipping any compiled packages there, like `Lwt` or any package that we choose to precompile. Having e.g. the top 100 OPAM packages there could be a significant speedup.

Reply:
Good point. I'll rename the repo to be more general to give us the option to add more packages to it in the future.